cf-reset-cache/reset-cache.py

```diff
@@ -24,20 +25,29 @@
     sys.exit(1)
 
 # get the paths from s3
-s3_conn = boto.connect_s3(access_key, access_secret)
-docs = s3_conn.get_bucket(bucket)
+s3_conn = boto3.client('s3', aws_access_key_id=access_key, aws_secret_access_key=access_secret)
+docs = s3_conn.list_objects(Bucket=bucket)
 items = []
 
-for key in docs.list():
-    index_file = "/index.html"
-    if key.name.endswith((index_file)):
+for key in docs['Contents']:
+    name = key['Key'].encode('utf-8')
+    index_file = "index.html"
+    if name.endswith((index_file)):
         # append the file without the postfix as well
-        items.append(key.name.replace(index_file, ""))
-        items.append(key.name.replace(index_file, "/"))
-    items.append(key.name)
-
-cf_conn = boto.connect_cloudfront(access_key, access_secret)
-inval_req = cf_conn.create_invalidation_request(cloudfront_dist, items)
+        items.append(name.replace(index_file, "/"))
+    items.append('/{}'.format(name))
```

Comment:
Seems like we need to prefix each file with a leading / now, so removed line 35 as it was causing an invalid InvalidationRequest error.